index.js

```diff
@@ -65,6 +86,18 @@
 
 // ***** Question 6 *****
 // *** Uncomment the lines below to test
+
+function printNamesAndPhones(users)
+{
+  for( let i = 0; i < users.length; i++)
+  {
+    console.log(users[i].name)
+    console.log(users[i].phones.cell)
+    console.log(users[i].phones.office)
+  }
+}
```

Comment:
Very good! Javascript has a couple other methods that may make it a little easier:

- Here's how it would look with for..of:
```javascript
 for (let user of users) {
  console.log(user.name);
  console.log(`Cell: ${user.phones.cell}`);
  console.log(`Office: ${user.phones.office}`);
 }
```

- Here's how it would look with forEach:
```javascript
 users.forEach(function(user) {
  console.log(user.name);
  console.log(`Cell: ${user.phones.cell}`);
  console.log(`Office: ${user.phones.office}`);
 })
```